README.md

```diff
@@ -1,5 +1,5 @@
+![](https://github.com/google/pprof/workflows/ci/badge.svg)
```

Comment:
The badge links to the image itself which is not useful and is uncommon to do. Please make the badge link to the CI status page, like the AppVeyor badge did and like Travis CI and Codecov badges do.

Reply:
Sent https://github.com/google/pprof/pull/549.